snappy/click.go

```diff
@@ -316,12 +326,12 @@ func (m *packageYaml) addPackageServices(baseDir string, inhibitHooks bool, inte
 		}
 		// If necessary, generate the DBus policy file so the framework
 		// service is allowed to start
-		if m.Type == snap.TypeFramework && service.BusName != "" {
-			content, err := genBusPolicyFile(service.BusName)
+		if m.Type == snap.TypeFramework && app.BusName != "" {
+			content, err := genBusPolicyFile(app.BusName)
```

Comment:
This is something that skills will interact with heavily. How can that method know about skills.Repository()?

This is not a blocker in any way, reading this makes me think about integration of skills into the overlord.